<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-DED1LDD" failures="0" tests="6" name="com.a2j.tests.Register_test" time="2.014" errors="1" timestamp="2020-05-14T20:46:58 IST" skipped="5">
  <testcase classname="com.a2j.tests.Register_test" name="role" time="0.000">
    <skipped/>
  </testcase> <!-- role -->
  <system-out/>
  <testcase classname="com.a2j.tests.Register_test" name="pannumber" time="0.000">
    <skipped/>
  </testcase> <!-- pannumber -->
  <system-out/>
  <testcase classname="com.a2j.tests.Register_test" name="panemail" time="0.000">
    <skipped/>
  </testcase> <!-- panemail -->
  <system-out/>
  <testcase classname="com.a2j.tests.Register_test" name="password" time="0.000">
    <skipped/>
  </testcase> <!-- password -->
  <system-out/>
  <testcase classname="com.a2j.tests.Register_test" name="confirmpass" time="0.000">
    <skipped/>
  </testcase> <!-- confirmpass -->
  <system-out/>
  <testcase classname="com.a2j.tests.Register_test" name="regbtn" time="2.014">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.a2j.tests.Register_test.regbtn(Register_test.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:63)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:348)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:619)
at org.testng.TestRunner.beforeRun(TestRunner.java:609)
at org.testng.TestRunner.run(TestRunner.java:580)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- regbtn -->
  <system-out/>
</testsuite> <!-- com.a2j.tests.Register_test -->
